.github/workflows/release.yml

```diff
@@ -677,7 +684,7 @@ jobs:
 
     steps:
       - name: Delete nightly release
-        uses: actions/github-script@v5
+        uses: actions/github-script@v6
         with:
           script: |
             const releases = await github.rest.repos.listReleases({
```

Comment:
You missed also bumping download-artifact to v3 in release.yml (in order to use actions with node 16 instead of node 12)